slack-jakarta-socket-mode-client/src/main/java/com/slack/api/jakarta_socket_mode/impl/JakartaSocketModeClientTyrusImpl.java

```diff
@@ -0,0 +1,435 @@
+package com.slack.api.jakarta_socket_mode.impl;
```

Comment:
I see this is almost an exact copy of `SocketModeClientTyrusImpl.java`, which is production worthy 🧠

Reply:
Yeah, you're right!